pyincore/analyses/epfdamage/epfdamage.py

```diff
@@ -256,7 +256,8 @@ def epf_damage_analysis_bulk_input(self, epfs, hazard_type, hazard_dataset_id,
                 liquefaction_prob = liquefaction_vals[i]['liqProbability']
 
                 selected_liq_fragility = liq_fragility_set[liq_epf_id]
-                pgd_limit_states = selected_liq_fragility.calculate_limit_state(liq_hazard_val, std_dev=std_dev)
+                pgd_limit_states = \
+                    selected_liq_fragility.calculate_limit_state_w_conversion(liq_hazard_val, std_dev=std_dev)
```

Comment:
is this change intentional? I can't find a test case to test EPF damage with liquefaction to confirm. I feel `calculate_limit_state_w_conversion()` method should be removed too. It is only being used in this one place